client/src/components/review.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ReviewText from './reviewText';
+import Report from './report';
+import HostResponse from './hostResponse';
+
+class Review extends React.Component {
+  render() {
+    const month = {
+      '01': 'January',
+      '02': 'February',
+      '03': 'March',
+      '04': 'April',
+      '05': 'May',
+      '06': 'June',
+      '07': 'July',
+      '08': 'August',
+      '09': 'September',
+      '10': 'October',
+      '11': 'November',
+      '12': 'December',
+    };
+    const {
```

Comment:
Interesting use of this.props and creating this class component. If you choose to keep this a class component, perhaps move month, date, formatDate and hostReponse into a constructor method's state. Because this component is presumably inheriting props from a parent component and does not appear to have any unique methods, perhaps make this a stateless component and add these variables above the return statement.

Reply:
Thanks for the feedback. I keep trying to make this a stateless component, but it keeps breaking every time I try due to the variables I am declaring. Where do you declare variables if it’s not in a class?